libmscore/edit.cpp

```diff
@@ -4666,22 +4669,43 @@ void Score::undoAddElement(Element* element)
             }
       }
 
+//---------------------------------------------------------
+//   nextChord
+//---------------------------------------------------------
+
+Chord* Score::nextChord(Segment* seg, Staff* staff)
```

Comment:
Hmmm, would a function like this be better off somewhere like `utils.cpp`? I'm not too bothered, since where we put `Score::x` functions is a bit of a mess anyway, but it would be good to consider it.